docs/sources/mimir/configure/configure-experimental-promql-functions.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Configure experimental PromQL functions
+description: Control what experimental PromQL functions are enabled for your Mimir installation.
+weight: 110
+---
+
+# Configure experimental PromQL functions
+
+Experimental PromQL functions are disabled by default in Mimir. They can be selectively enabled per-tenant or globally for all users.
+All functions can be enabled using the special function name `all` or each individual function can be enabled specifically.
+
+You can enable functions globally for all users using the main Mimir configuration file.
+
+```yaml
+limits:
+  enabled_promql_experimental_functions: info,sort_by_label,sort_by_label_desc
+```
+
+You can enable functions for specific tenants using [per-tenant overrides]({{< relref "./about-runtime-configuration" >}}):
```

Comment:
We should use the full URL here instead of a relref.

Reply:
I'm going to leave this as-is in this PR since it seems like all the other docs in `configure/` are using relref. We can update them all at once.